src/Admin/Form/RoleType.php

```diff
@@ -14,15 +16,39 @@
 
 class RoleType extends AbstractType
 {
+    private PluginService $pluginService;
+
+    public function __construct(PluginService $pluginService)
```

Comment:
use constructor property promotion and make readonly

`__construct(private readonly PluginService $pluginService)`